### ISSUE TITLE
Expect-CT: put max-age first

### ISSUE THREAD

**Issue by EvanHahn** [helmetjs/helmet#225]
It's a little weird that `max-age` comes second after `enforce`:

https://github.com/helmetjs/helmet/blob/936cd27c91364a91d108a9fe31b81c92fb423338/middlewares/expect-ct/index.ts#L24-L28

We should reorder these lines and update the tests.

**Comment by CyberZujo** [comment]
```
 directives.push(`max-age=${parseMaxAge(options.maxAge)}`);

  if (options.enforce) {
    directives.push("enforce");
  }
```
Did you mean this? Is there any constraint for the max age or just literally to reorder lines?

**Comment by EvanHahn** [comment]
Yup, that's it!

You will probably need to update the tests, too.

**Comment by CyberZujo** [comment]
` it("can enable enforcement", async () => {
    await check(expectCt({ enforce: true }), {
      "expect-ct": "enforce, max-age=123",
    });
  });

  it("can explicitly disable enforcement", async () => {
    await check(expectCt({ enforce: false }), {
      "expect-ct": "max-age=123",
    });
  });
`
I've found these cases and the previous value of max-age was set to 0, probably now since it's reordered, I have to provide a value, so is that okay or, it is in 

`test/expect-ct.test.ts`

You can assign this to me and I'll make a PR.

**Comment by EvanHahn** [comment]
I'd make the change and then run `npm test` to see what breaks, and then fix those tests. At least one of the tests you showed should change, though it's possible that there are others.

I can't assign this issue to you (GitHub doesn't allow me to) but feel free to make a pull request.

**Comment by EvanHahn** [comment]
@CyberZujo I'm planning to release the next major version of Helmet on Sunday, August 2 (a few days from now). Would you mind waiting until then? No problem if not, but I think it will be less work if we wait.

**Comment by CyberZujo** [comment]
Yes, no problem, I'll wait until then and then I'll make the PR.

**Comment by EvanHahn** [comment]
Helmet 4 (and `expect-ct@1.0.0`) was just released, so feel free to go ahead with your change. No rush, though!

**Comment by EvanHahn** [comment]
Closed by #264.
